packages/react/src/index.ts

```diff
@@ -89,7 +83,15 @@ export const useSetChain = (
   const [settingChain, setInProgress] = useState<boolean>(false)
 
   const [connectedChain, setConnectedChain] = useState<ConnectedChain | null>(
-    null
+    () => {
+      const initialWallets = (web3Onboard as OnboardAPI).state.get().wallets
+      return (
+        (
+          initialWallets.find(({ label }) => label === walletLabel) ||
+          initialWallets[0]
+        )?.chains[0] || null
```

Comment:
Can you remove the optional chaining (?.) when you have a moment?
We have to stick with a bit older ES operators in order to support all of our customers bundles and compilers